snuba/datasets/errors_replacer.py

```diff
@@ -485,7 +485,11 @@ def process_message(self, message: ReplacementMessage) -> Optional[Replacement]:
         type_ = message.action_type
 
         if type_ in REPLACEMENT_EVENT_TYPES:
-            metrics.increment("process", 1, tags={"type": type_})
+            metrics.increment(
```

Comment:
Is this the only metric which is relevant per consumer group?

Reply:
I believe so. The other metrics already have consumer group data.